src/SQLScripts/auto-growth-zero.sql

```diff
@@ -0,0 +1,15 @@
+select mf.database_id
+      ,mf.[file_id]
+      ,mf.[type]
+      ,mf.[type_desc]
+      ,mf.[state]
+      ,mf.state_desc
+      ,mf.growth
+      ,mf.is_percent_growth
+      ,sizeMb=(try_cast(mf.size as bigint)*8192.)/power(1024,2) 

```

Comment:
- Mb vs MB
- Also what's with the square brackets in some cases and not others

Reply:
* Mb = camelCase('MB') but point taken. snake_case it is!
* square brackets are non-reserved keywords